frontends/web/src/routes/account/send/send-wrapper.tsx

```diff
@@ -29,14 +32,40 @@ type TSendProps = {
 }
 
 export const SendWrapper = ({ accounts, code, deviceIDs, devices }: TSendProps) => {
+  const { t } = useTranslation();
   const { defaultCurrency } = useContext(RatesContext);
+
+  const [bb01Paired, setBB01Paired] = useState<boolean>();
+  const [noMobileChannelError, setNoMobileChannelError] = useState<boolean>();
+
   const account = findAccount(accounts, code);
+  const product = deviceIDs.length > 0 ? devices[deviceIDs[0]] : undefined;
+
+  useEffect(() => {
+    if (account && product && product === 'bitbox') {
+      const fetchData = async () => {
+        try {
+          const mobileChannel = await hasMobileChannel(product)();
```

Comment:
I get a 404 :(

<img width="998" alt="Screenshot 2024-11-25 at 16 40 13" src="https://github.com/user-attachments/assets/c6a288f6-bbb3-4999-ae8c-2bfc08d7ca63">

Reply:
oh, it is wrong to pass `product` to `hasMobileChannel` not sure why I did that. I'll update it.

Reply:
same for `getDeviceInfo` actually